README.md

```diff
@@ -7,62 +7,94 @@ products:
 
-Chatbots are an easy way to provide answers to frequently asked questions by users. However, most chatbots fail to engage with users in a meaningful way because there is no human in the loop when the chatbot fails.  
+Chatbots are an easy way to provide answers to frequently asked questions by end-users. However, most chatbots fail to engage with end-users in a meaningful way because there is no human in the loop when the chatbot fails. 
 
-FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is unable to help. One can ask the bot a question and the bot responds with an answer if it's in the knowledge base. If not, the bot allows the user to submit a query which then gets posted in a pre-configured team of experts who are help to provide support by acting upon the notifications from within their Team itself. 
+FAQ Plus bot is a friendly Q&A bot that brings a human in the loop when it is unable to help. One can ask the bot a question and the bot responds with an answer if it's in the knowledge base. If not, the bot allows the end-user to submit a query which then gets posted in a pre-configured team of experts to provide support by acting upon the notifications from within their team itself.
 
-**FAQ Plus has following features:**
+**The July 2020 (version 3) release of FAQ Plus includes a multi-turn feature to the end user experience. With the multi-turn feature, users will be presented with follow-up options along with an answer to their question. This enables the FAQ Plus bot to answer the user's question with more relevance. Multi-turn follow-up options are programmed directly into the QnA Maker when the tenant admin uploads the proper documents into the knowledge base.**
 
-An end-user interacting with FAQ Plus:
+**Earlier releases of FAQ Plus has following features:**
 
-![FAQ Plus in action (user view)](https://github.com/OfficeDev/microsoft-teams-apps-faqplusv2/wiki/Images/FAQPlusEndUser.gif)
+*	An Enduser interacting with FAQ Plus:
 
-Experts team using FAQ Plus:
+![FAQ Plus in action (user view)](https://github.com/OfficeDev/microsoft-teams-faqplusplus-app/wiki/images/FAQPlusEndUser.gif)
 
-![FAQ Plus in action (experts view)](https://github.com/OfficeDev/microsoft-teams-apps-faqplusv2/wiki/Images/FAQPlusExperts.gif)
+*	Experts team using FAQ Plus:
 
+![FAQ Plus in action (experts view)](https://github.com/OfficeDev/microsoft-teams-faqplusplus-app/wiki/images/FAQPlusExperts.gif)
 
-### FAQ Plus [Version 2] adds the following new capabilities
 
-Members of the experts team will now be able to
-- add a new QnA pair to the knowledge base directly using messaging extension
-- edit and delete QnA pairs added through the bot
-- track history of experts updating QnA pair
-- configure the answer to show up as an adaptive card with additional details like title, subtitle, image, etc. instead of usual text based answer
+**FAQ Plus [Version 2] (To_Do_Link) provides new features to experts team such as:**

```

Comment:
Let's consistently remove the [v2] tag.
Have we tagged the v1 repo as deprecated (or the equivalent), and direct users to this repo? We should do that before merging in this change.

Reply:
Removed the v2 tag.
@abbodh , can you please help in tagging v1 repo as deprecated.